### PR TITLE
[DOCS] Fix typo in RoutingNode comment

### PR DESCRIPTION
**Pull request by howardhuanghua**
Fix RoutingNode typo.

### CODE REVIEW

File: server/src/main/java/org/elasticsearch/cluster/routing/RoutingNode.java [hunk 1]
```diff
@@ -144,7 +144,7 @@ void add(ShardRouting shard) {
     void update(ShardRouting oldShard, ShardRouting newShard) {
         assert invariant();
         if (shards.containsKey(oldShard.shardId()) == false) {
-            // Shard was already removed by routing nodes iterator
+            // Shard has already been removed by routing nodes iterator
```
Comment (jrodewig) [inline server/src/main/java/org/elasticsearch/cluster/routing/RoutingNode.java:147]:
```suggestion
            // Shard was already removed by routing nodes iterator
```

File: server/src/main/java/org/elasticsearch/cluster/routing/RoutingNode.java [hunk 2]
```diff
@@ -122,7 +122,7 @@ public int size() {
 
     /**
      * Add a new shard to this node
-     * @param shard Shard to crate on this Node
+     * @param shard Shard to be created on this Node
```
Comment (jrodewig) [inline server/src/main/java/org/elasticsearch/cluster/routing/RoutingNode.java:125]:
```suggestion
     * @param shard Shard to create on this Node
```

**Review by jrodewig — APPROVED** [review]
LGTM. I reverted some of the changes as they were fine as-is.

**Comment by jrodewig** [comment]
@elasticmachine test this please

**Review by MrHeer — APPROVED** [review]

**Comment by jrodewig** [comment]
@elasticmachine test this please

**Comment by jrodewig** [comment]
@elasticmachine update branch

**Comment by jrodewig** [comment]
@elasticmachine run elasticsearch-ci/bwc

**Comment by jrodewig** [comment]
@elasticmachine run elasticsearch-ci/default-distro

**Comment by jrodewig** [comment]
@elasticmachine run elasticsearch-ci/bwc

**Comment by howardhuanghua** [comment]
Hi @jrodewig Shall I need to merge the lastest master code?

**Comment by howardhuanghua** [comment]
Hi @jrodewig I have done the latest master code merge just now.

**Comment by jrodewig** [comment]
Thanks @howardhuanghua.

I'll run the test CI again and merge assuming everything passes.

**Comment by jrodewig** [comment]
@elasticmachine test this please

**Comment by jrodewig** [comment]
Merged! Thanks again for your contribution, @howardhuanghua.

**Comment by jrodewig** [comment]
### Backport commits
master d747c1b
7.x affc395

**Pull request merged**